SofaKernel/modules/SofaDefaultType/src/sofa/defaulttype/typeinfo/DataTypeInfoDynamicWrapper.h

```diff
@@ -77,8 +77,8 @@ class DataTypeInfoDynamicWrapper : public AbstractTypeInfo, public Info
 
     static AbstractTypeInfo* get() { static DataTypeInfoDynamicWrapper<Info> t; return &t; }
 
-    virtual const TypeInfoId& getBaseTypeId() const { return TypeInfoId::GetTypeId<typename Info::BaseType>(); }
-    virtual const TypeInfoId& getValueTypeId() const { return TypeInfoId::GetTypeId<typename Info::ValueType>(); }
+    virtual const TypeInfoId& getBaseTypeId() const override { return TypeInfoId::GetTypeId<typename Info::BaseType>(); }
```

Comment:
Maybe remove the virtual keyword while being here? It is redundant if you put the override.

Reply:
can do